pkg/controllers/edgeconnect/secret/secret.go

```diff
@@ -80,3 +92,17 @@ func safeEdgeConnectCfg(cfg config.EdgeConnect) string {
 
 	return string(safe)
 }
+
+func addDefaultServiceAccount() (config.Secret, error) {
+	newToken, err := dttoken.New("dt0e01")
```

Comment:
How is it made sure, that excactly this token is also used in the connection settings object?

Reply:
yeah, it's dependency on another PR. will discuss with @albertogdd